Bottom: 4e37d1a24ca55fc17f8a096b813467c85eec8c90
Top:    e7e4570d276f7db9553cb6c5a9bff34a403a8808
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:31:12 +0000

Refresh of select-c-set-an-explicit-flag-

---

diff --git a/src/select.c b/src/select.c
index 4cdc008..ca6a660 100644
--- a/src/select.c
+++ b/src/select.c
@@ -959,6 +959,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
   END_LOOP;
   if (Type & NET_TYPE)
     {
+      SaveFindFlag (FOUNDFLAG);
       InitConnectionLookup ();
       changed = ResetConnections (true) || changed;
 
@@ -983,6 +984,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       changed = SelectConnection (Flag) || changed;
       changed = ResetConnections (false) || changed;
       FreeConnectionLookupMemory ();
+      RestoreFindFlag ();
     }
 
 #if defined(HAVE_REGCOMP)
